### PR TITLE
bounty_requests: auto-email anyone who’s done a funded request in the…

### LINKED ISSUE #2419: auto-email to anyone who’s done a funded request in the past from vivek.singh
anyone who has successfully done a gitcoin request in the past should be auto - emailed from vivek.singh@consensys.net email every 2 months with the following copy:

We should be careful to make sure we don't email someone more than **once every 2 months** (i.e. if they make 3 Gitcoin Requests in August, they should still only get one email in October) 

```
Howdy @{username}, 

This is Vivek from Gitcoin.  I noticed you made a funded Gitcoin Requests a few months ago and just wanted to check in.  How'd it go?  Any feedback for us?

Let us know if you have any bounties in your near future -- we'll pay attention to [Gitcoin Requests](https://gitcoin.co/requests/) from you as we know you've suggested good things in the past 🙂 

Best,
V
```

pls copy https://github.com/gitcoinco/web/blob/master/app/retail/emails.py#L122-L205 to see how we do this in the codebase already

# success criteria:

1. management command created
2. management command scheduled in crontab
3. management command generates the above email, according to specs

### PR DESCRIPTION
**Pull request by pinkiebell**
… past from vivek.singh

- Introduces the `bounty_requests_email` mgmt command.
- Additional BountyRequestMeta db storage class helper to keep track of
mails
- crontab entry

Fixes #2419

### CODE REVIEW

File: app/marketing/mails.py [hunk 1]
```diff
@@ -793,3 +793,34 @@ def new_funding_limit_increase_request(profile, cleaned_data):
...
+        setup_lang(to_email)
+        subject = _('Bounty Request Feedback')
+        body = f'Howdy @{profile.username},\n\n'\
+               'This is Vivek from Gitcoin. '\
+               'I noticed you made a funded Gitcoin Requests '\
+               'a few months ago and just wanted to check in. '\
+               'How\'d it go? Any feedback for us?\n\n'\
+               'Let us know if you have any bounties in your near future '\
+               '-- we\'ll pay attention to '\
+               '[Gitcoin Requests](https://gitcoin.co/requests/) '\
```
Comment (owocki) [inline app/marketing/mails.py]: @vs77bb is this the right copy?

 i think we might want to remove the markdown from this..

File: app/bounty_requests/management/commands/bounty_requests_email.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+class Command(BaseCommand):
+
+    help = 'Dispatches email feedback for funded Bounty Requests at most once in 8 weeks.'
+
+    def handle(self, *args, **options):
+        now = timezone.now()
+        delta = timedelta(weeks=8)
+        bounty_requests = BountyRequest.objects.filter(
+            status=BountyRequest.STATUS_FUNDED,
+            created_on__lte=now - delta,
```
[the remark this answers is not in the document]
Comment (pinkiebell) [reply app/bounty_requests/management/commands/bounty_requests_email.py]: @owocki 
This just filters the Bounty Requests, the actual check is further down ( BountyRequestMeta)

File: app/bounty_requests/management/commands/bounty_requests_email.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        )
+
+        for request in bounty_requests:
+            profile = request.requested_by
+
+            if not profile or not profile.email:
+                continue
+
+            meta = BountyRequestMeta.objects.filter(profile=profile).last()
```
[the remark this answers is not in the document]
Comment (pinkiebell) [reply app/bounty_requests/management/commands/bounty_requests_email.py]: Can also be `first()` but the idea here is that we only have one `BountyRequestMeta` per `Profile`.

File: app/marketing/mails.py (hunk 1 above)
Comment (owocki) [reply app/marketing/mails.py]: @vs77bb

**Comment by owocki** [comment]
code LGTM -- didnt test tho

**Pull request merged**